sunpy/net/dataretriever/client.py

```diff
@@ -9,6 +9,14 @@
 from collections import OrderedDict, namedtuple
 from functools import partial
 
+# Remove in 1.0
```

Comment:
can you modify this comment to start with `python 3:` as I have been using that in a few places for post-0.9 grepping.